src/e2e/README.md

```diff
@@ -0,0 +1,49 @@
+# How to Run e2e Test
+
+###
+
+#### Install npm packages error will show if node 18.12.0^ is not installed, `nvm install 18.12.0` should fix this.
+
+```sh
+npm install
+```
+
+#### Install playwright along with necessary browser agents
+
+```sh
+npx playwright install
+```
+
+#### Generate .env from .env-dist
+
+```sh
+cp .env-dist .env
+```
+
+#### Add some additional values to the .env file
+
+`E2E_TEST_ACCOUNT_EMAIL=`
+
+`E2E_TEST_ACCOUNT_PASSWORD=`
+
+`E2E_TEST_BASE_URL=`
```

Comment:
might need a comment on what value we expect here. I assume a FQDN, but not sure about trailing slashes, etc.

I was able to search and find a couple examples:
- https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/src/e2e/utils/helpers.js#L22
- https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/playwright.config.js#L57
- https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/src/e2e/utils/helpers.js#L13-L17

Reply:
will just remove this, as choosing the E2E_TEST_ENV will update this env var